ibm/service/power/resource_ibm_pi_volume_group.go

```diff
@@ -35,51 +35,78 @@ func ResourceIBMPIVolumeGroup() *schema.Resource {
 			Delete: schema.DefaultTimeout(10 * time.Minute),
 		},
 		Schema: map[string]*schema.Schema{
-			helpers.PICloudInstanceId: {
-				Type:        schema.TypeString,
-				Required:    true,
-				Description: "Cloud Instance ID - This is the service_instance_id.",
+			// Arguments
+			Arg_CloudInstanceID: {
+				Description:  "The GUID of the service instance associated with an account.",
+				Required:     true,
+				Type:         schema.TypeString,
+				ValidateFunc: validation.NoZeroValues,
 			},
-			PIVolumeGroupName: {
-				Type:          schema.TypeString,
+			Arg_ConsistencyGroupName: {
+				ConflictsWith: []string{Arg_VolumeGroupName},
+				Description:   "The name of consistency group at storage controller level",
 				Optional:      true,
-				Description:   "Volume Group Name to create",
-				ConflictsWith: []string{PIVolumeGroupConsistencyGroupName},
-			},
-			PIVolumeGroupConsistencyGroupName: {
 				Type:          schema.TypeString,
+			},
+			Arg_VolumeGroupName: {
+				ConflictsWith: []string{Arg_ConsistencyGroupName},
+				Description:   "Volume Group Name to create",
 				Optional:      true,
-				Description:   "The name of consistency group at storage controller level",
-				ConflictsWith: []string{PIVolumeGroupName},
+				Type:          schema.TypeString,
 			},
-			PIVolumeIds: {
-				Type:        schema.TypeSet,
-				Required:    true,
+			Arg_VolumeIDs: {
+				Description: "List of volumes to add in volume group",
 				Elem:        &schema.Schema{Type: schema.TypeString},
+				Required:    true,
 				Set:         schema.HashString,
-				Description: "List of volumes to add in volume group",
+				Type:        schema.TypeSet,
 			},
 
-			// Computed Attributes
-			"volume_group_id": {
-				Type:        schema.TypeString,
+			// Attributes
+			Attr_ConsistencyGroupName: {
 				Computed:    true,
-				Description: "Volume Group ID",
-			},
-			"volume_group_status": {
+				Description: "Consistency Group Name if volume is a part of volume group",
 				Type:        schema.TypeString,
-				Computed:    true,
-				Description: "Volume Group Status",
 			},
-			"replication_status": {
-				Type:        schema.TypeString,
+			Attr_ReplicationStatus: {
 				Computed:    true,
 				Description: "Volume Group Replication Status",
+				Type:        schema.TypeString,
 			},
-			"consistency_group_name": {
+			Attr_StatusDescriptionErrors: {
```

Comment:
Can you please add `status_description_errors` in docs?

Reply:
added description